From 63edb0dcbeec16f00d5a9e54e80e1a0a5b2d6488 Mon Sep 17 00:00:00 2001
From: Marek Kasik <mkasik@redhat.com>
Date: Mon, 20 Jun 2011 15:02:39 +0000
Subject: Printers: Include cups/ppd.h explicitly

cups/ppd.h is not included by cups/cups.h since CUPS 1.5.

Upstream: yes, removed the configure changes
---
diff --git a/panels/printers/pp-new-printer-dialog.c b/panels/printers/pp-new-printer-dialog.c
index a6164fa..d972c4d 100644
--- a/panels/printers/pp-new-printer-dialog.c
+++ b/panels/printers/pp-new-printer-dialog.c
@@ -31,6 +31,7 @@
 #include <gtk/gtk.h>
 
 #include <cups/cups.h>
+#include <cups/ppd.h>
 
 #include "pp-new-printer-dialog.h"
 #include "pp-utils.h"
diff --git a/panels/printers/pp-utils.c b/panels/printers/pp-utils.c
index 07ccfc3..faa29b9 100644
--- a/panels/printers/pp-utils.c
+++ b/panels/printers/pp-utils.c
@@ -25,6 +25,7 @@
 #include <glib/gstdio.h>
 #include <gtk/gtk.h>
 #include <cups/cups.h>
+#include <cups/ppd.h>
 #include <dbus/dbus-glib.h>
 
 #include "pp-utils.h"
--
cgit v0.9.0.2
